study.md

```diff
@@ -31,7 +31,9 @@ manipulates it by sorting it and then reversing it in place (i.e., modifying the
 reference to the original).
 
 ```ruby
-# your answer here
+Array = [23, 56, 3, 7]
+Array.sort.reverse
```

Comment:
Since we want to the modify the array in place, the invocations of `sort` and `reverse` should be `sort!` and `reverse!`, respectively.  Also, you should not reassign `Array`.